frontend/src/components/UserStoryDescriptions.vue

```diff
@@ -4,7 +4,7 @@
       <b-list-group-item
         v-for="(story, idx) of userStories"
         v-show="idx === index"
-        :key="story.name"
+        :key="story.position"
```

Comment:
Why don't you use the property `id` of userStories or use the index `idx` as key?